libs/scalajslib/src/mill/scalajslib/worker/ScalaJSWorker.scala

```diff
@@ -258,7 +260,11 @@ private[scalajslib] class ScalaJSWorker(jobs: Int)
 @internal
 private[scalajslib] object ScalaJSWorkerExternalModule extends mill.api.ExternalModule {
 
-  def scalaJSWorker: Worker[ScalaJSWorker] =
-    Task.Worker { new ScalaJSWorker(Task.ctx().jobs) }
+  def scalaJSWorker: Worker[ScalaJSWorker] = Task.Worker {
+    new ScalaJSWorker(
+      jobs = Task.ctx().jobs,
+      linkerJobs = 2
```

Comment:
Perhaps this value could be exposed on `ScalaJSModuleAPI`? It can have a nice low default but allow people to tweak it to their needs or based on some environmental heuristic? Just a suggestion.

Exposing it on the API also slightly improves the transparency around what's going on here, but perhaps this will need to be documented somehow?

Reply:
Yeah, I already thought about how to configure it, but didn't want to overengineer it. 

The natural place for a config task would be the `ScalaJSWorker`, which is currently not designed to be customized, in a way other worker are, for example the `JvmWorkerModule`. Also, since there are potentially more that one `ScalaJSWorker`, we would need to introduce a new shared worker, so this route isn't a trivial change.

What would be somewhat easier is accepting an environment variable. 

Also, we should converge to a "sensible default". I don't work with Scala.JS often, so I have no "feeling" for what a good value might be. We might also apply some logic based on heuristics, which I don't have.